qrenderdoc/Code/Interface/Extensions.h

```diff
@@ -908,7 +908,24 @@ passed nothing will happen.
 :param QWidget combo: The combo box.
 :param List[str] options: The new options for the combo box.
 )");
-  virtual void SetComboOptions(QWidget *combo, const rdcarray<rdcstr> &options) = 0;
+  virtual void SetComboBoxOptions(QWidget *combo, const rdcarray<rdcstr> &options) = 0;
+
+  DOCUMENT(R"(Get the number of options in a drop-down combo box. If another type of widget is
+passed ``0`` will be returned.
```

Comment:
Minor note, this is not true as it looks like the implementation (as well as the existing implementation for `SetComboOptions` which has a bug) will crash if another widget is passed. The function needs to check that the return of `qobject_cast` is non-NULL.